.vscode/settings.json

```diff
@@ -12,4 +12,7 @@
   "python.testing.pytestEnabled": true,
   "pylint.importStrategy": "fromEnvironment",
   "python.linting.lintOnSave": true,
+  "files.associations": {
+    "*.yaml": "home-assistant"
```

Comment:
This change is incorrect. Please exclude it from the PR.

Reply:
deleted